Ginger/GingerCoreCommon/Repository/GingerSolutionRepository.cs

```diff
@@ -66,6 +67,8 @@ public static SolutionRepository CreateGingerSolutionRepository()
 
             SR.AddItemInfo<RemoteServiceGrid>("*.Ginger.RemoteServiceGrid.xml", SolutionRepository.cSolutionRootFolderSign + "RemoteServiceGrid", true, "RemoteServiceGrid", PropertyNameForFileName: nameof(RemoteServiceGrid.Name));
 
+            SR.AddItemInfo<AccessibilityConfiguration>("*.Ginger.Configuration.xml", SolutionRepository.cSolutionRootFolderSign + "Configurations", true, "AccessibilityConfiguration", PropertyNameForFileName: nameof(AccessibilityConfiguration.Name));
```

Comment:
Please check if we load RQM or Jira settings, same folder is being used and new one is not getting created

Reply:
Done